README.md

```diff
@@ -1,6 +1,6 @@
 ## About
 `appmap-python` is a Python package for recording
-[AppMaps](https://github.com/applandinc/appmap) of your code. "AppMap" is a data format
+[AppMaps](https://github.com/applandinc/appmap) of your code. `AppMap` is a data format
```

Comment:
The punctuation here was the same as https://github.com/applandinc/appmap-ruby#about . Maybe they should be consistent?

Reply:
OK, I'll make them consistent, thank you for the review Alan!